cloud_formation/configs/cachedb.py

```diff
@@ -213,10 +215,8 @@ def create_config(session, domain, keypair=None, user_data=None):
         ['s3:GetObject', 's3:PutObject'],
         { 'AWS': role})
 
-    creating_tile_bucket = False
```

Comment:
Why is this taken out?

Reply:
It’s no longer used. PyFlake flagged it.